server/run.py

```diff
@@ -61,7 +61,7 @@ def sort():
                         "confident score": 0})
         parameters = json.dumps({'classifier_ids': [classifier_id]})
         url_result = visual_recognition.classify(images_file=images_file,
-                                                 parameters=parameters)
+                                                 parameters=parameters).get_result()
```

Comment:
Thanks for pull request, can you move this line under 79 characters to follow the flake8 guideline? Thank you.